src/components/App.js

```diff
@@ -58,14 +57,20 @@ class App extends Component {
           <Scene key="login" component={Login}
             initial={!isAndroid} hideNavBar={true} type={ActionConst.RESET}/>
 
-          <Scene key="main" component={Main} title="Bridgeme"
+          {/* The right button(filter) function will be added later */}
+          <Scene key="main" component={Main} title="Bridgeme" rightTitle="right"
+            rightButtonTextStyle={{ color: 'transparent', }}
+            rightButtonIconStyle={{ marginBottom: 13, marginRight: 6 }}
```

Comment:
Why there is a comma after color at line 62, but not after marginRight at line 63?